projects/kit/components/routable-dialog/routable-dialog.component.ts

```diff
@@ -1,40 +1,51 @@
-import {ChangeDetectionStrategy, Component, inject, INJECTOR} from '@angular/core';
+import {ChangeDetectionStrategy, Component, inject, INJECTOR, Type} from '@angular/core';
 import {ActivatedRoute, Router} from '@angular/router';
 import {TuiDestroyService} from '@taiga-ui/cdk';
 import {TuiDialogService} from '@taiga-ui/core';
 import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
 import {takeUntil} from 'rxjs';
 
 @Component({
+    standalone: true,
```

Comment:
Add schematic migration, please

Reply:
If I've not mistaken, schematic should just delete mention of `TuiRoutableDialogModule`

Reply:
another PR